substrate/runtime/primitives/src/traits.rs

```diff
@@ -35,7 +35,7 @@ pub trait Verify {
 }
 
 /// Ed25519 signature verify.
-#[derive(Eq, PartialEq, Clone)]
+#[derive(Eq, PartialEq, Clone, Default)]
 #[cfg_attr(feature = "std", derive(Debug, Serialize))]
 pub struct Ed25519Signature(H512);
```

Comment:
this isn't a trait, and makes the type alias `Signature` which references this confusing

Reply:
it's not meant to be a trait. not sure where `Signature` is being used, but it should be removed in favour of this.

Reply:
I mean that it's in the `traits` module

Reply:
i see